Source/Core/DolphinQt/Config/Mapping/WiimoteEmuMetroid.cpp

```diff
@@ -95,6 +159,15 @@ void WiimoteEmuMetroid::CreateMainLayout()
       GetPort(), WiimoteEmu::WiimoteGroup::Visors));
   groupbox1->addWidget(visor_box);
 
+  auto* morphball_control_box = CreateGroupBox(tr("Morphball Profile"), Wiimote::GetWiimoteGroup(
+    GetPort(), WiimoteEmu::WiimoteGroup::MorphballControls));
+  groupbox1->addWidget(morphball_control_box);
+  m_morphball_combobox = (morphball_control_box->findChild<QComboBox*>(tr("ProfileList")));
+  m_morphball_combobox->setToolTip(tr("Set the controller profile to use\nwhen in Morph Ball in MP3."));
+  m_morphball_combobox->setEditable(false);
```

Comment:
The 'in MP3' part can be removed if added to all three wii games.

Reply:
Oh right.  I hadn't thought about Trilogy.  Been so focused on Prime 3, lol.